### PR TITLE
CLDN-1045 Pipeline for superset-ui

### PR DESCRIPTION
**Pull request by cccs-nik**

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -0,0 +1,69 @@
+trigger:
+  branches:
+    include:
+      - cccs-*
+      - feature/CLDN-*
```
Comment (cccs-tom) [inline azure-pipelines.yml:5]: In general, we don't include feature branches in the trigger. All PRs will be automatically built, so when devs are ready to build they can create a PR.

File: azure-pipelines.yml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+      displayName: 'Build packages'
+    - script: |
+        yarn test
+      displayName: 'Run unit tests'
+  - job: PublishArtifactsToFeed
+    dependsOn: BuildNpmArtifacts
+    steps:
+    - template : publish-template.yml
+      parameters:
+        workingDirs: [
```
Comment (cccs-tom) [inline azure-pipelines.yml:37]: This JSON-style list is kind of weird in a YAML file. Can it be converted to YAML?

File: azure-pipelines.yml [hunk 3]
```diff
@@ -0,0 +1,69 @@
+trigger:
+  branches:
+    include:
+      - cccs-*
+      - feature/CLDN-*
+
+pool:
+  vmImage: 'ubuntu-latest'
```
Comment (cccs-tom) [inline azure-pipelines.yml:8]: I'm a big fan of using quotes only when they're required in YAML
```suggestion
  vmImage: ubuntu-latest
```

File: azure-pipelines.yml [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+variables:
+  tag: '$(Build.BuildId)'
```
Comment (cccs-tom) [inline azure-pipelines.yml:11]:
```suggestion
  tag: $(Build.BuildId)
```

File: azure-pipelines.yml [hunk 5]
```diff
@@ -0,0 +1,69 @@
...
+stages:
+- stage: Build_and_Publish_Npm_Artifacts
+  displayName: Build, package and publish npm artifacts.
+  jobs:
+  - job: BuildNpmArtifacts
+    displayName: Build, package and publish npm artifacts.
+    steps:
+    - task: NodeTool@0
+      inputs:
+        versionSpec: '14.x'
```
Comment (cccs-tom) [inline azure-pipelines.yml:22]:
```suggestion
        versionSpec: 14.x
```

**Review by cccs-tom — CHANGES_REQUESTED** [review]

**Review by cccs-tom — APPROVED** [review]

**Review by cccs-RyanS — APPROVED** [review]

**Pull request merged**